Code/docker.js

```diff
@@ -1704,6 +1707,37 @@ define([
                 }
             }
 
+            // on keyup for .wcFrameButtonBar > .wcFrameButton
+            function __onKeyUpPanelButton(e) {
+                if(e.which === 13 || e.which === 32) {
+                    $('body').removeClass('wcDisableSelection');
```

Comment:
Looking at the code I am not able to figure the reason for this operation.
Please add required comments for the same.